runners/spark/2/src/main/java/org/apache/beam/runners/spark/structuredstreaming/SparkStructuredStreamingRunner.java

```diff
@@ -111,22 +109,8 @@ public static SparkStructuredStreamingRunner create(
    * @return A pipeline runner that will execute with specified options.
    */
   public static SparkStructuredStreamingRunner fromOptions(PipelineOptions options) {
-    SparkStructuredStreamingPipelineOptions sparkOptions =
```

Comment:
We were executing this relatively heavy step twice because we were doing it in `fromOptions` and in each Spark runner `.run()` method so I removed it from here and it is now up to the runner to take care of it.